src/redismodule.h

```diff
@@ -257,6 +257,10 @@ typedef uint64_t RedisModuleTimerID;
 /* Declare that the module can handle diskless async replication with RedisModule_SetModuleOptions. */
 #define REDISMODULE_OPTIONS_HANDLE_REPL_ASYNC_LOAD    (1<<2)
 
+ /* Declare that the module want to get nested key space notifications.
+  * If enabled, the module is responsible to break endless loop. */
+ #define REDISMODULE_OPTIONS_ALLOW_NESTED_KEYSPACE_NOTIFICATIONS    (1<<3)
```

Comment:
https://github.com/redis/redis/issues/11537

Reply:
i'd argue that it is related to this PR since it adds a new option and a a compatibility issue.
it's real quick to add it, i don't see a need for another PR or issue.

Reply:
@oranagra added to this PR and updated the top comment.